sources/models/shared-defaults/oci-defaults-docker-cri.toml

```diff
@@ -0,0 +1,10 @@
+[metadata.settings.oci-defaults]
+affected-services = ["oci-defaults", "docker"]
+
+[services.oci-defaults]
+configuration-files = ["oci-defaults"]
+restart-commands = []
+
+[configuration-files.oci-defaults]
+path = "/etc/docker/base-spec.json"
```

Comment:
In the commit message, you refer to this file as `cri-base.json`. Please adjust.